.github/config.yml

```diff
@@ -0,0 +1,18 @@
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  Thanks for opening your first issue here! Be sure to follow the issue template!
```

Comment:
Is it practical to hyperlink the template?

Reply:
Well, this message should appear after they picked the template. And the template is already populated in the issue.